documentation/Kubecheck.md

```diff
@@ -317,6 +318,13 @@ This test checks if defined package repositories are available from nodes.
 
 This test checks if needed package are available from nodes.
 
+###### 015 Kernel version
+
+*Task*: `software.kernel.version`
+
+This test checks the Linux kernel version for a bad kernel version, if it is equal to a bad one, it issues a warning
+
```

Comment:
Let's add something like this:
'bad kernel version' is a kernel with detected serious issue that affects any part of cluster, therefore it's unsupported

Reply:
Update doc